posixlib/src/main/scala/scala/scalanative/runtime/SocketHelpers.scala

```diff
@@ -77,7 +77,7 @@ object SocketHelpers {
         }
 
         val sentBytes = send(sock, toCString("echo"), 4, 0)
-        if (sentBytes < 4) {
+        if (sentBytes.toLong < 4) {
```

Comment:
We do seem to have an overload for `<` for comparing `Word` and `Int`, why this explicit conversion needed here?